3.interpret_models/log_reg_plate_3_3p_cp_renorm_fs_data/feature_importances/nbconverted/seperate_compartments.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Here the features are seperated according to compartments for later analysis
+
+# In[1]:
+
+
+# %%--%%| <qGnYViiwRD|SyZ3qa8iz3>
+r"""°°°
+## Imports
+°°°"""
+
+
+# In[2]:
+
+
+import pandas as pd
+from pathlib import Path
+import itertools
```

Comment:
```suggestion
import itertools
import pandas as pd
from pathlib import Path
```
Following python's pep8 [imports rule](https://peps.python.org/pep-0008/#imports)

Reply:
Changed!